package.json

```diff
@@ -254,7 +254,8 @@
         ],
         "coverageReporters": [
             "lcov"
-        ]
+        ],
+        "testTimeout": 30000
```

Comment:
issue: this is not needed, the tests are fast enough to use the standard timeout.